### PR TITLE
Kafka 16887 : Upgrade the document to add remote copy/fetch quotas metrics values.

### PR DESCRIPTION
**Pull request by Nancy-ksolves**
Upgrade the document to add remote copy/fetch quotas metrics values.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/ops.html [hunk 1]
```diff
@@ -1931,6 +1931,27 @@ <h4 class="anchor-heading"><a id="tiered_storage_monitoring" class="anchor-link"
         <td>Average idle percent of thread pool for copying data to remote storage</td>
         <td>kafka.log.remote:type=RemoteLogManager,name=RemoteLogManagerTasksAvgIdlePercent</td>
       </tr>
+      <tr>
+        <td>RemoteLogManager Avg Broker Throttled Fetch Time</td>
```
Comment (abhijeetk88) [inline docs/ops.html]: Should we have this something like - `RemoteLogManager Avg Broker Fetch Throttle Time`

File: docs/ops.html [hunk 2]
```diff
@@ -1931,6 +1931,27 @@ <h4 class="anchor-heading"><a id="tiered_storage_monitoring" class="anchor-link"
...
+        <td>The average time in millis remote fetches was throttled by a broker</td>
+        <td>kafka.server:type=RemoteLogManager, name=remote-fetch-throttle-time-avg	</td>
+      </tr>
+      <tr>
+        <td>RemoteLogManager Max Broker Throttled Fetch Time</td>
+        <td>The max time in millis remote fetches was throttled by a broker</td>
+        <td>kafka.server:type=RemoteLogManager, name=remote-fetch-throttle-time-max	</td>
+      </tr>
+      <tr>
+        <td>RemoteLogManager Avg Broker Throttled copies Time</td>
```
Comment (abhijeetk88) [inline docs/ops.html]: And similarly - `RemoteLogManager Avg Broker Copy Throttle Time`

**Comment by Nancy-ksolves** [comment]
Hi @abhijeetk88, Mentioned changes has been done. Kindly review it again.

**Comment by abhijeetk88** [comment]
LGTM

**Review by showuon — APPROVED** [review]
LGTM!

**Review by abhijeetk88 — APPROVED** [review]

**Review by satishd — APPROVED** [review]
Thanks @Nancy-ksolves for the PR. LGTM.

**Comment by satishd** [comment]
There are a few unrelated test failures, merging it to trunk.

**Pull request merged**